spec.bs

```diff
@@ -1187,6 +1235,101 @@ To <dfn>report win</dfn> given a [=leading bid info=] |leadingBidInfo|, a [=stri
   1. TODO: Store |resultUrl| and |reportingBeaconMap| in the {{FencedFrameConfig}} as appropriate.
 </div>
 
+# K-anonymity # {#k-anonymity}
+  
+Two goals of this specification rely on applying k-anonymity thresholds:
+  
+  * To prevent cross-site leaks:  Inputs to event-level reporting functions, `reportWin()` and
+    `reportResult()`, only contain limited cross-site information. As described in
+    [[#privacy-considerations]], part of this limiting is done by
+    applying k-anonymity requirements to the ad URLs.
+  * To prevent microtargeting:  The browser applies k-anonymity requirements on the ad URLs to
+    ensure that the same ad or ad component is being shown to at least some minimum number of
+    people.
+
+The browser enforces these k-anonymity requirements by maintaining counts of how many times each
+ad and ad component has been shown to users. These counts are maintained across users, so the counting must
+be done on a central <dfn>k-anonymity server</dfn>. This specification relies on two operations to query and
+increment the counts: [=query k-anonymity count=] and [=increment k-anonymity count=].
+  
+The details of how the [=k-anonymity server=] is operated and accessed are [=implementation-defined=]
+but it should be done in a way that prevents the server operator from joining the identity of two
+query or increment requests. One way to help prevent this is by making accesses to the server go
+through an HTTP proxy that prevents the server from seeing the browsers' IP addresses.
+
+The browser should choose a <dfn>k-anonymity threshold</dfn>, otherwise known as the value for "k", 
+and a <dfn>k-anonymity duration</dfn> depending
+on the projected sizes of interest groups and their privacy goals. For example an implementation

```

Comment:
```suggestion
on the projected sizes of interest groups and the browser's privacy goals. For example an implementation
```

I think?

Reply:
Ya, I figured "their" would refer to it, but I'll go with your suggestion as it's clearer.